frontend/packages/client/src/hooks/useLeaderBoard.js

```diff
@@ -2,28 +2,27 @@ import { useQuery } from 'react-query';
 import { useErrorHandlerContext } from '../contexts/ErrorHandler';
 import fetchLeaderBoard from 'api/leaderboard';
 
-export default function useLeaderBoard({ communityId = 0 } = {}) {
+export default function useLeaderBoard({ communityId = 0, addr = '' } = {}) {
   const { notifyError } = useErrorHandlerContext();
   const { isLoading, isError, data, error } = useQuery(
-    ['leaderboard'],
-    async () => fetchLeaderBoard(communityId)
+    ['leaderboard', addr],
+    async () => fetchLeaderBoard(communityId, addr)
   );
 
   if (isError) {
     notifyError(error);
   }
 
+  const users = data?.data?.users ?? [];
+  const currentUser = data?.data?.current_user;
```

Comment:
could this be like 
`const currentUser = data?.data?.currentUser;` 
so it's camelcase?

Reply:
Upated.